flake8.rc

```diff
@@ -0,0 +1,19 @@
+[flake8]
+max_line_length = 80
+ignore =
+    # Ignore extra blank lines
+    E302,
```

Comment:
I suggest to enable E302. It does not hurt, once you stick to the blank line rules.